internal/ingress/controller/controller.go

```diff
@@ -543,6 +549,8 @@ func (n *NGINXController) getBackendServers(ingresses []*ingress.Ingress) ([]*in
 						loc.Port = ups.Port
 						loc.Service = ups.Service
 						loc.Ingress = ing
+
+						loc.PathType = path.PathType
```

Comment:
Given the break at line 533 this line will be reached iff `*loc.PathType == *path.PathType`, so that means this line is redundant right?

Reply:
Removed